packages/dashboard/src/components/broadcastsTable.tsx

```diff
@@ -0,0 +1,94 @@
+import { DataGrid, GridColDef } from "@mui/x-data-grid";
+import { useRouter } from "next/router";
+import React from "react";
+
+import { useAppStore } from "../lib/appStore";
+import { monospaceCell } from "../lib/datagridCells";
+
+interface Row {
+  id: string;
+  name: string;
+  updatedAt: string;
+  triggeredAt: string;
+}
+
+const baseColumn: Partial<GridColDef<Row>> = {
+  flex: 1,
+  sortable: false,
+  filterable: false,
+  renderCell: monospaceCell,
+};
+
+export default function BroadcastsTable() {
+  const router = useRouter();
+  const broadcasts = useAppStore((store) => store.broadcasts);
+
+  const broadcastsRow: Row[] = [];
+
+  broadcasts.forEach((broadcast) => {
+    const row: Row = {
+      id: broadcast.id,
+      name: broadcast.name,
+      updatedAt: new Date(broadcast.updatedAt).toISOString(),
+      triggeredAt: broadcast.triggeredAt
+        ? new Date(broadcast.triggeredAt).toISOString()
+        : "Not Triggered",
+    };
+    broadcastsRow.push(row);
+  });
+
+  return (
+    <DataGrid
+      rows={broadcastsRow}
+      sx={{
+        height: "100%",
+        width: "100%",
+        ".MuiDataGrid-row:first-child": {
```

Comment:
You seem to be re-using the same styles over and over. Can you deduplicate this code?

Reply:
Ok sure, shall I use this code to override the theme for DataGrid in `themeCustomization`?

Reply:
I think that's too complicated IMO. You could just export a value from a shared `components/resourceTable.tsx` file

```typescript
const RESOURCE_TABLE_STYLE: SxProps<Theme> = {
...
}
```

Reply:
Ok sure, will do it!